### ISSUE TITLE
Not grouping (multiple sheets)

### ISSUE THREAD

**Issue by hildogjr** [hildogjr/KiCost#97]
I am creating a purchase list using a KiCad schematic with hierarchical sheets. But KiCost is not grouping the equal parts of the different sheets (it is possible to see this be the 100nF capacitor that all them have the same footprint).

Even, I think is necessary to improve the grouping code to group the subparts of different designators.

Or components "with different values" (see the SW601-606 in the XML to understand better, or the AI701-AI704, GND101, all these tests pins with the name as the corresponding signal).

[acquire-PWM.xml.zip](https://github.com/xesscorp/KiCost/files/1470501/acquire-PWM.xml.zip)

**Comment by hildogjr** [comment]
I found the source of the issue.
Each component group is using a different library, because of the migration to new KLC and I used different `libpart` (library cache file) to each of the sheets. This that generate also the issue of not group equal manufacture parts / code in a multi part designator case (e.g. different connector could be have different quantities of a same object, as pin or cable).
I know that I could change the libraries of this project to re-group correctly, but this will not fix the mult parts case.
What do you think @xesscorp, will this need a change in the `componet_groups` generated in `kicad.py` or added some command in `kicost` call to ignore the footprint?

**Comment by hildogjr** [comment]
Also will be interesting merge different `desc` parts keeping the `desc`information in the spreadsheet. E.g. `desc = description1; description2`.

**Comment by hildogjr** [comment]
Related with a new issue #102, closed this to keep the discussion centralized.

**Comment by hildogjr** [comment]
Remains after #102 improvement and others...
Use https://github.com/xesscorp/KiCost/blob/master/tests/acquire-PWM.xml to compliance.
